lib/manageiq/appliance_console/prompts.rb

```diff
@@ -113,6 +113,27 @@ def ask_for_password(prompt, default = nil)
       pass == "********" ? (default || "") : pass
     end
 
+    def ask_for_new_password(prompt, default: nil, allow_empty: false, retry_limit: 1, confirm_password: true)
+      count = 0
+      loop do
+        password1 = ask_for_password(prompt, default)
+        if password1.strip.empty? && !allow_empty
+          say("\nPassword can not be empty, please try again")
+          next
+        end
+
+        return password1 if password1 == default || !confirm_password
+
+        password2 = ask_for_password("#{prompt} again")
```

Comment:
I know this "again" came from one of the examples, but I think this would be a nicer presentation:

```suggestion
        password2 = ask_for_password("Re-enter #{prompt}")
```

Reply:
:+1: changed

Reply:
@Fryguy `just_ask` prepends `"Enter the "` to the prompt, so when I have it this way we end up with
```
Enter the database password on localhost: *******
Enter the Re-enter database password on localhost: *****
```

Reply:
Ugh - that's annoying :(

Would prefer if it was 

```
Enter the database password on localhost: *******
Re-enter the database password on localhost: *****
```

If we can't do that it's fine - we can go back to the "again" text - sorry for the runaround.

Reply:
Yeah so the cluster database just printed the same string twice, prefer that or the "again" suffix?
Longer term I started working on just_ask taking a prefix argument which defaults to "Enter the " but we could replace that with Re-enter.  It is all positional arguments right now though so need to check all of the callers and make sure I don't break something

Reply:
Yeah I was thinking of a parameter to just_ask also...can definitely be a followup.  I think putting the "#{original_text} again" text is fine for this PR.